src/content/docs/tutorial-innovation-and-growth/innovate-with-new-relic.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Improve your code and release qualty with New Relic"
+metaDescription: "Our development quality guide helps increase stability by using New Relic to ensure that fewer defects are introduced into the code base."
+redirects: 
+  - /docs/new-relic-solutions/observability-maturity/innovation-growth/development-quality-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/innovation-growth/improving-development-quality-guide
+  - /docs/new-relic-solutions/observability-maturity/innovation-growth/development-release-quality-guide
+---
+
+import journeyDiagram from 'images/inno_diagram_improvement-pipeline.webp'
+
+High quality code and high quality releases are crucial for an organization to innovate and grow. Poor quality code diverts engineering time that could use to innovate, instead wasting it on troubleshooting and fixing. High quality releases will help keep a steady userbase by keeping customers happy. Poor quality releases and poor quality code both have a strong negative impact on customer satisfaction and revenue.
+
+This tutorial helps you identify specific key performance indicators and processes that drive stability through code and release quality improvement. It focuses on improving the quality of your development first, then improving the quality of your releases. 
+
+<img
+    title="Improvement Journey"
+    alt="A diagram showing the four parts of improving your development and release quality with New Relic: gather KPIs, improve your development quality, improve your release quality, innovate and grow"
+    src={journeyDiagram}
+/>
+
+You're a good candidate for using this guide if:
+* You're not currently measuring code quality.
+* Your code quality is perceived as poor.
+* You don't understand where your developers are spending their time.
+* Your organization suffers from too many outages due to application defects.
+
+## Before you begin [#getting-started]
+
+You should be familiar with the New Relic platform before you can use it to improve your development quality. If you're totally new to our platform, no worries! You can take [this New Relic University course](https://learn.newrelic.com/overview-course) to get you up to speed fast. It's also a good idea to familarize yourself with the following:
+
+* [New Relic platform fundamentals](https://learn.newrelic.com/fundamentals-course)
+* [Introduction to instrumentation](https://learn.newrelic.com/self-paced-intro-to-instrumentation-with-new-relic)
+* [New Relic dashboards and NRQL](https://learn.newrelic.com/webinar-getting-started-with-dashboards-nrql)
+* [New Relic alerting best practices](https://learn.newrelic.com/webinar-new-relic-alerts-fundamentals)
+
+<DocTiles>
```

Comment:
A bit confusing -- are one of these meant to be 'previous'? A bit confusing to have both here leading to 'next'. Might make the reader think they have to return to this page to 'work through' the other doc correctly

Reply:
Agreed, since this is a step by step series I removed the second tile as it's required to do the first one anyway. Thank you!